server/index.js

```diff
@@ -6,14 +7,16 @@ const apiPort = 3000;
 const db = require('./db');
 const posts = require('./routes/post.routes');
 const auth = require('./routes/auth.routes');
+const images = require('./routes/image.routes');
 const requireAuth = require('./middlewares/requireAuth');
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(cors());
-app.use(bodyParser.json());
+app.use(bodyParser.json({ limit: '5mb' }));
```

Comment:
i choose this arbitrary limit for payload size, this is the max payload including pictures. also the UI have an error if you exceed.